ext/libxml/tests/bug76777.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+Bug #76777 (first parameter of libxml_set_external_entity_loader callback undefined)
+--SKIPIF--
+<?php if (!extension_loaded('libxml')) die('skip'); ?>
```

Comment:
The test should also be skipped if `--offline` is given, i.e. something like `if (getenv("SKIP_ONLINE_TESTS")) die('skip online test');` would be reasonable.